Userland/Utilities/userdel.cpp

```diff
@@ -36,7 +36,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto& target_account = account_or_error.value();
 
     if (remove_home)
-        TRY(Core::System::unveil(target_account.home_directory(), "c"sv));
+        TRY(Core::System::unveil(target_account.home_directory(), "r"sv));
```

Comment:
Wait, but removal of files should still require `w`, no? :thonk:

Reply:
OpenBSD's `unveil` uses `c` (pledge's `cpath`) for creation and removal of the unveiled paths. I guess the issue is that we're trying to access the path before unlinking which would require `r`?

Reply:
It seems we only need cpath and veil `r` in order to `sys$rmdir` and `sys$unlink`.

If that's a bug in the Kernel, then I guess that should be fixed first :D